package.json

```diff
@@ -34,7 +34,7 @@
     "ts-node": "^3.2.0",
     "tslint": "^5.7.0",
     "typedoc": "^0.8",
-    "typescript": "^2.7.1",
+    "typescript": "2.7.2",
```

Comment:
please keep the caret, yarn.lock will lock the version anyway

Reply:
it was if ppl were not using yarn